src/theme/mixins.scss

```diff
@@ -0,0 +1,54 @@
+@mixin mobile {
+  // TODO: Decide on breakpoint values. maybe get it from Carbon
```

Comment:
since we don't use carbon in this project, probably it's better to remove these TODO comments